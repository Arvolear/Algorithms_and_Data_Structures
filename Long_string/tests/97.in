94070307561488235988056474144
348510039523101336316594