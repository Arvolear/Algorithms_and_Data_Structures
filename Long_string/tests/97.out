94070656071527759089392790738
94069959051448712886720157550
32784446606204563615366258683885359868578607759145536
269921